src/main/java/com/salesforce/storm/spout/dynamic/filter/StaticMessageFilter.java

```diff
@@ -60,18 +68,18 @@ public boolean equals(Object other) {
 
         StaticMessageFilter that = (StaticMessageFilter) other;
 
-        return uniqueId.equals(that.uniqueId);
+        return id.equals(that.id);
     }
 
     @Override
     public int hashCode() {
-        return uniqueId.hashCode();
+        return id.hashCode();
     }
 
     @Override
     public String toString() {
```

Comment:
I wonder if toString should just return the id here?

Reply:
Done, probably not a big deal honestly as this is just a test class.